public/main.js

```diff
@@ -40,15 +51,35 @@ $jobSearch.addEventListener('submit', () => {
 $createUser.addEventListener('submit', () => {
   event.preventDefault()
   const formData = new FormData($createUser)
-  post('/users', formData)
+  if (window.location.hash === '#profile/create') {
+    post('/users', formData)
+      .then(() => {
+        $createUser.reset()
+        renderSelectUsers()
+        window.location.href = '#profile'
```

Comment:
Control the `location.hash` from your `router` instance. Extend the class if needed.